opencog/reduct/reduct/contin_reduction.cc

```diff
@@ -31,107 +31,127 @@
 // Note: the rule names are useful when debugging the reduct engine
 // (uncomment META_RULE_DEBUG in meta_rules.cc)
 
-namespace opencog { namespace reduct {
+namespace opencog
+{
+namespace reduct
+{
+
+const rule &contin_reduction(int reduct_effort,
+                             const vertex_set &ignore_ops)
+{
+	return setup_contine_reduction(reduct_effort, ignore_ops);
```

Comment:
There's a typo here `setup_contine_reduction` -> `setup_contin_reduction`. I know you didn't introduce it but could I ask you to fix it?